massa-execution-worker/src/execution.rs

```diff
@@ -1338,33 +1354,16 @@ impl ExecutionState {
         debug!("execute_final_slot: execution started");
         let exec_out = self.execute_slot(slot, exec_target, selector);
 
+        // update versioning stats
+        // Note: This update the MIP Store and this must be done before apply_final_execution_output
+        //       as it will write the final state (and thus the MIP store) to the DB
+        self.update_versioning_stats(&exec_out.block_info, slot);
```

Comment:
this could be called at the beginning of apply_final_execution_output to unify everything

Reply:
done